### DISCUSSION TITLE
How i can hide date from list and single posts

### DISCUSSION THREAD

**Discussion by ahfpak** [chipzoller/hugo-clarity#382] [Q&A] [answered]
i want to hide date from single posts and list of homepage.. where i can change the code

**Comment by rootwork** [comment, marked as answer]
Hi @ahfpak -- it's in the https://github.com/chipzoller/hugo-clarity/blob/master/layouts/partials/post-meta.html file -- see the section in `<span class="post_date">` (The `<span>` just before it is the calendar icon, which you might also want to remove.)

If you put a copy of this in your local `layouts/partials/` directory, that will override that partial from Hugo Clarity with your customizations.

**Reply by ahfpak** [reply]
Thank-you-very-much.
This-worked-for-me.
